mk/e2e.new.mk

```diff
@@ -140,7 +140,7 @@ test/e2e-kubernetes: $(E2E_DEPS_TARGETS)
 	$(MAKE) test/e2e/k8s/stop/cluster/kuma-1
 
 .PHONY: test/e2e-universal
-test/e2e-universal: build/kumactl images/test k3d/network/create
+test/e2e-universal: $(E2E_DEPS_TARGETS) k3d/network/create
```

Comment:
that would require building ALL the images, which is a waste of time. We only need `images/test`

Reply:
Ha ok gotcha. E2E_DEPS_TARGETS is required for KM but I'll do something else then! Thx for poiting out